source/uwp/winui3/WinUI3ObjectModelTest/Package.appxmanifest

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="22a2b2d3-1de6-4c6b-b954-7523c603f0ae"
+    Publisher="CN=krschau"
```

Comment:
>krschau [](http://example.com/codeflow?start=18&length=7)

please change to Microsoft Corporation as well as the PublisherDisplayName

Reply:
Looking at the other Publishers in the various uwp Package.appxmanifest's, it looks like most of them are "AdaptiveCards", one is "Microsoft Corporation", and one had a person's alias like mine. I'll change the other alias too (unless I shouldn't), but do we prefer "Microsoft Corporation" or "AdaptiveCards"?